Automattic/sensei
# Add Module column to Lessons table

Addresses #907.

## Testing
- Browse to the _Lessons_ page.
- Ensure that any module that has been assigned to a lesson is listed in the _Module_ column.
- Click on a module link. You should be taken to the _Edit Module_ page.
- Return to the _Lessons_ screen. Go to _Screen Options_ and deselect the _Module_ checkbox.
- Ensure the _Module_ column is no longer visible.
